ivy/functional/frontends/torch/reduction_ops.py

```diff
@@ -31,3 +31,10 @@ def any(input, dim=None, keepdim=False, *, out=None):
     if ivy.is_uint_dtype(input_dtype):
         ret = ivy.astype(ret, input_dtype, out=out)
     return ret
+
+
+def nansum(input, dim=None, keepdim=False, *, out=None):
```

Comment:
args should be identical to native: https://pytorch.org/docs/stable/generated/torch.nansum.html

Reply:
`torch.nansum(input, *, dtype=None)` is just a shorter form of longer `torch.nansum(input, dim=None, keepdim=False, *, dtype=None)`.

Reply:
ahh sorry, I probably overlooked last time, my bad! Will re-review asap!